core/time/at_spec.rb

```diff
@@ -135,4 +135,11 @@
       lambda { Time.at(Time.now, 500000) }.should raise_error(TypeError)
     end
   end
+
+  describe "passed BigDecimal" do
+    require 'bigdecimal'
+    it "doesn't round input value" do
+      Time.at(BigDecimal.new('1.1')).to_f.should == 1.1
+    end
+  end
```

Comment:
Could you move it in the first describe `"passed Numeric"` since BigDecimal < Numeric?